test/neo/main.d

```diff
@@ -290,6 +293,61 @@ class Test : Task
 
         enforce!("==")(received_count, records_written);
     }
+
+    /***************************************************************************
+
+        Runs a simple test where a single record is serialized, written to the
+        node with Put, fetched with Get, then deserialized.
+
+    ***************************************************************************/
+
+    private void testSerialize ( )
+    {
+        struct Str
```

Comment:
Test deserializing instance of a versioned struct using `MultiVersionDecorator` and non-versioned struct with `deserialize` overload accepting one?